package.json

```diff
@@ -72,6 +72,7 @@
         "cross-spawn": "^4.0.0",
         "dequeue": "^1.0.5",
         "event-kit": "^2.0.0",
+        "flow": "^0.2.3",
```

Comment:
I think you're looking for `flow-bin` instead - this is [flow](https://www.npmjs.com/package/flow)